src/Aside/Aside.js

```diff
@@ -55,8 +58,9 @@ class Aside extends Component {
                   isInvalidCardName={this.state.isInvalidCardName}
                   hasDuplicates={this.state.hasDuplicates}/>
         <AsideBody cardNames={this.state.cardNames} 
-        // card names is objects, need array of strings
-                    removeListItem={this.removeListItem}/>
+                   removeListItem={this.removeListItem}
+                   setCardCount={this.setCardCount}
+                   userCards={this.state.userCards} />
```

Comment:
So you're passing userCards down as props in order to make changes to it in cardListItem?